modules/extensions-support/src/main/java/org/mule/module/extension/internal/model/property/SubtypesModelProperty.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.model.property;
+
+
+import org.mule.extension.api.introspection.ModelProperty;
+import org.mule.module.extension.internal.util.IntrospectionUtils;
+import org.mule.util.CollectionUtils;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
+/**
+ * An immutable model property which specifies the declared implementations
```

Comment:
this javadoc doesn't really explain what this does. You can just say that it holds the mapping declared on the annotation and link it